ui/themes/app.js

```diff
@@ -1,5 +1,5 @@
-import { createTheme } from '@material-ui/core/styles';
-import { blueGrey } from '@material-ui/core/colors';
+import { createTheme } from '@mui/material/styles';
```

Comment:
@SAHU-01 @sudhanshutech aren't we supposed to migrate to the Sistent theme which is already mui v5? // @nebula-aac

Reply:
Yes, @Yashsharma1911 I had a discussion about this with @sudhanshutech  and  @aabidsofi19 yesterday, I'll be closing this PR. And migrate pages/components to use sistent theme, once that is done, this theme migration won't be necessary.